.github/workflows/ci.yml

```diff
@@ -33,10 +33,10 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
 
-    - name: Install packages
-      run: |
-        sudo apt update
-        sudo apt install texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended texlive-lang-all
+    - uses: awalsh128/cache-apt-pkgs-action@v1
+      with:
+        packages: texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended texlive-lang-all
+        version: 1.0
```

Comment:
Do we need `execute_install_scripts: true` ?

Reply:
The install seemed to worked fine without it hence I didn't add it. These packages probably don't have any post install scripts.